frontends/web/src/routes/account/send/send.tsx

```diff
@@ -166,6 +166,18 @@ class Send extends Component<Props, State> {
           break;
         }
         break;
+      case 'backend':
```

Comment:
small nit: To me `backend` doesn't make sense as everything that is pushed is coming from the backend. (same reason I think we should not have `api/backend`)

maybe `account` would be more suitable?

Reply:
ignore, I forgot that it is already like that.. :P